sdk/storage/azure-storage-file-share/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release History
 
+## Version X.X.X (XXXX-XX-XX)
```

Comment:
The general guidance here is to put the specific version we have in the repo followed by (unreleased) see https://github.com/Azure/azure-sdk/blob/master/docs/policies/releases.md#changelog-guidance

Reply:
We haven't run the version update script for this feature branch yet, so the pom still reads 12.0.0. Because we branched a couple weeks ago, we're behind on some of these updates to master. I think our goal is to get the code for the features into the feature branch and then we will rebase and update these sort of infrastructural things for the whole feature branch at once.

Reply:
I highly suggest you update the version in this branch as well. It is never a good idea to be building the same version as what is published as they will cause conflicts in different modes and you will never know which one you are getting. 

cc @JimSuplizio

Reply:
Created a PR here to update versions in our feature branch https://github.com/Azure/azure-sdk-for-java/pull/6981